src/AryUtil.chpl

```diff
@@ -1044,7 +1080,7 @@ module AryUtil
       // index -> order for the input array's indices
       // e.g., order = k + (nz * j) + (nz * ny * i)
       inline proc indexToOrder(idx: rank*?t): t
-      where (t==int) || (t==uint(64)) {
+        where (t==int) || (t==uint(64)) {
```

Comment:
I think you might want to fix the alignment below this too, unless the linter is giving you trouble

Reply:
I was specifically refering to the indentation below this of what is inside the `where`

Reply:
I didn't  write that but I think we should have a team discussion about what indentation we should use.  I see throughout the code very commonly that the where statement is indented to the same levels as the code.

Reply:
Discussed and agreed on format in our Tuesday meeting.